frontend/public/scss/receipt.scss

```diff
@@ -0,0 +1,257 @@
+// sass-lint:disable mixins-before-declarations
+$label-color: #8a8b8c;
+
+.receipt-wrapper {
+
+  padding: 20px 15px;
+  margin: 0 auto;
+  max-width: 1018px;
+  font: 500 18px/23px 'Rajdhani', Times New Roman, serif;
```

Comment:
Can this `font` rule (and where it appears elsewhere in this file - there's 11 total occurrences of Rajdhani in the file) be removed? This should use the standard fonts from the base layout.scss.